beacon-chain/rpc/validator/server_test.go

```diff
@@ -27,9 +28,15 @@ import (
 	"github.com/prysmaticlabs/prysm/shared/testutil/assert"
 	"github.com/prysmaticlabs/prysm/shared/testutil/require"
 	"github.com/prysmaticlabs/prysm/shared/trieutil"
+	"github.com/sirupsen/logrus"
 	logTest "github.com/sirupsen/logrus/hooks/test"
 )
 
+func init() {
```

Comment:
I believe a better place for this will be in `validator_test` (where we have `TestMain`, it should actually go into `TestMain` function)

Reply:
Will do!

Reply:
To clarify: instead of using `init()`, just put log configuring statements into `validator_test:TestMain()`.

Reply:
Fixed